syncmaster/backend/api/v1/connections.py

```diff
@@ -80,13 +81,16 @@ async def read_connections(
     if pagination.items:
         credentials = await unit_of_work.credentials.read_bulk([item.id for item in pagination.items])
         items = [
-            ReadConnectionSchema(
-                id=item.id,
-                group_id=item.group_id,
-                name=item.name,
-                description=item.description,
-                auth_data=credentials.get(item.id, None),
-                data=item.data,
+            TypeAdapter(ReadConnectionSchema).validate_python(
+                {
+                    "id": item.id,
+                    "group_id": item.group_id,
+                    "name": item.name,
+                    "description": item.description,
+                    "type": item.type,
+                    "data": item.data,
+                    "auth_data": credentials.get(item.id, None),
+                },
             )
```

Comment:
Calling ReadConnectionSchema constructor doesn't work anymore with these changes?

Reply:
Yes, it throws `TypeError: 'types.UnionType' object is not callable` otherwise